packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -10,67 +10,224 @@
+
+  if (window.location.pathname) {
+    redirect_uri += window.location.pathname;
+  }
+}
 
-[
-  'access-token',
-  'invitee'
-].forEach((id) => {
-  const el = document.getElementById(id);
 
-  el.value = localStorage.getItem(id);
-  el.addEventListener('change', (event) => {
-    localStorage.setItem(id, event.target.value);
+// OAuth section
+if (window.location.hash) {
+  // Hash is created which gives us the access token,
+  // which will be used in the SDK initialization process
+  initializeWebexOAuthVersion();
+}
+
+// Initialize Webex to authorize through OAuth
+function initializeWebexOAuthVersion() {
+  // Get's redirect URI for OAuth option
+  getOAuthRedirectURI();
+
+  // eslint-disable-next-line no-multi-assign
+  webex = window.webex = Webex.init({
+    config: {
+      logger: {
+        level: 'debug'
+      },
+      meetings: {
+        reconnection: {
+          enabled: true
+        }
+      },
+      // Any other sdk config we need
+      credentials: {
+        client_id: 'C7c3f1143a552d88d40b2afff87600c366c830850231597fb6c1c1e28a5110a4f',
+        redirect_uri,
+        scope: 'spark:all spark:kms'
+      }
+    }
   });
-});
+}
 
-// There's a few different events that'll let us know we should initialize
-// Webex and start listening for incoming calls, so we'll wrap a few things
-// up in a function.
-function connect() {
-  return new Promise((resolve) => {
-    if (!webex) {
-      // eslint-disable-next-line no-multi-assign
-      webex = window.webex = Webex.init({
-        config: {
-          logger: {
-            level: 'debug'
-          },
-          meetings: {
-            reconnection: {
-              enabled: true
-            }
-          }
-          // Any other sdk config we need
-        },
+// Creates OAuth Form
+function createOAuthForm() {
+  // Present and hide correct form
+  document.getElementById('token-auth').style.display = 'none';
+  document.getElementById('oauth').style.display = 'block';
+
+  // Now, we adding event handler to login button to initiate login when clicked
+  document.getElementById('login').addEventListener('click', (event) => {
+    // Let's make sure we don't reload the page when we submit the form
+    event.preventDefault();
+
+    // Initialize the Webex
+    initializeWebexOAuthVersion();
 
-        credentials: {
-          access_token: document.getElementById('access-token').value
+    // Initiate login to Webex account
+    webex.authorization.initiateLogin();
+  });
+
+  // Now, we adding event handler to login out button to initiate login out when clicked
+  document.getElementById('logout').addEventListener('click', (event) => {
+    // Let's make sure we don't reload the page when we submit the form
+    event.preventDefault();
+
+    // Redirects user to sign out page
+    webex.logout();
+  });
+
+  // Immediately enabled all button in the oauth section when Webex is ready to be used
+  webex.once('ready', () => {
```

Comment:
When I load the page, this immediately throws an error saying `webex` is not defined.